atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,292 @@
+            properties = {
+                "competency_id": competency_id,
+                "name": competency["properties"]["name"],
+                "text": competency["properties"]["text"],
+                "cluster_id": clusters[best_medoid_idx]["properties"]["cluster_id"],
+                "cluster_similarity_score": similarity_score[best_medoid_idx]
+            }
+            self.weaviate_client.update_property_by_id(CollectionNames.COMPETENCY.value, competency["id"], properties)
+
+        for index in range(len(texts)):
+            text_entry = texts[index]
+            competency = self.weaviate_client.get_embeddings_by_property(CollectionNames.COMPETENCY.value, "cluster_id",
```

Ruff (0.11.9)

Line too long (120 > 88)

(E501)